### DISCUSSION TITLE
Not detecting Acurite Temp Sensors (06002M)

### DISCUSSION THREAD

**Discussion by iointerrupt** [arendst/Tasmota#16872] [Q&A] [answered]
### PROBLEM DESCRIPTION
I have 2 Sonoff RF Bridges R2 (white) with latest Tasmota Sensors builds.  My Acurite 06002M/00592TXR temp sensor does not show up in the raw readings.  My RTL-SDR using rtl_433 has no issues with capturing the sensor readings.  All my other RF 433mhz sensors are picked up fine by the bridge.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic):  **Sonoff RF Bridges R2** (2 of them)
- [x] Tasmota binary firmware version number used: **12.2.0(sensors)**
  - [x] **Pre-compiled**
  - [ ] Self-compiled
- [x] Flashing tools used: Standard Sonoff RF Bridge 2 flashing using USB FTDI (no cuttings).
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
{"NAME":"Generic","GPIO":[32,3200,1,3232,1,1,0,0,1,320,1,0,0,0],"FLAG":0,"BASE":25}
{"Module":{"25":"Sonoff Bridge"}}
{"GPIO0":{"32":"Button1"},"GPIO1":{"3200":"Serial Tx"},"GPIO2":{"0":"None"},"GPIO3":{"3232":"Serial Rx"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"0":"None"},"GPIO13":{"320":"Led_i1"},"GPIO14":{"0":"None"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua


```
- [x] Provide the output of this command: `Status 0`:
```lua
{"Status":{"Module":25,"DeviceName":"rfbridge2","FriendlyName":["rfbridge2"],"Topic":"rfbridge2","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":0,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0}}
{"StatusPRM":{"Baudrate":19200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota.bin.gz","RestartReason":"Software/System restart","Uptime":"0T13:18:15","StartupUTC":"2022-10-21T02:35:15","Sleep":50,"CfgHolder":4617,"BootCount":33,"BCResetTime":"2022-06-11T04:47:15","SaveCount":95,"SaveAddress":"FA000"}}
{"StatusFWR":{"Version":"12.2.0(sensors)","BuildDateTime":"2022-10-17T08:41:56","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8285","CR":"375/699"}}
{"StatusLOG":{"SerialLog":0,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["4F5V2ZXA124-1d",""],"TelePeriod":300,"Resolution":"558180E0","SetOption":["80008009","2805C80001000600003C5A0A192800000000","00000000","00006000","00004000","00000000"]}}
{"StatusMEM":{"ProgramSize":687,"Free":316,"Heap":15,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"14325E","FlashFrequency":40,"FlashMode":"DOUT","Features":["00000809","8F9A8787","0415A005","B7F7BFCF","05DA9BC4","64367CC7","00084052","20000000","54000020","00000001"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,14,16,17,20,21,24,29,34,62","Sensors":"1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,17,18,19,20,21,22,26,28,31,34,37,39,40,42,43,45,51,52,55,56,58,59,62,64,66,67,74,98"}}
{"StatusNET":{"Hostname":"rfbridge2","IPAddress":"<REDACTED>","Gateway":"<REDACTED>","Subnetmask":"255.255.255.0","DNSServer1":"<REDACTED>","DNSServer2":"<REDACTED>","Mac":"<REDACTED>","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
{"StatusMQT":{"MqttHost":"<REDACTED>","MqttPort":1883,"MqttClientMask":"rfbridge2_486EB4","MqttClient":"rfbridge2_486EB4","MqttUser":"iot","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
{"StatusTIM":{"UTC":"2022-10-21T15:53:30","Local":"2022-10-21T10:53:30","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"-05:00","Sunrise":"01:19","Sunset":"11:49"}}
{"StatusSNS":{"Time":1666367610}}
{"StatusSTS":{"Time":1666367610,"Uptime":"0T13:18:15","UptimeSec":47895,"Heap":15,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"<REDACTED>","BSSId":"<REDACTED>","Channel":1,"Mode":"11n","RSSI":100,"Signal":-42,"LinkCount":1,"Downtime":"0T00:00:05"}}}


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
The sensor emits data (led blinks periodically during emit) but Tasmota does not show any results

### EXPECTED BEHAVIOUR
Raw results would be broadcast via mqtt or show up in console

### SCREENSHOTS
N/A

### ADDITIONAL CONTEXT
N/A

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment, marked as answer]
So the Sonoff R2 is not compatible with the protocol used by those sensors

You may want to read the thread about Sonoff RF Bridge R2 to see suggestion on hardware modification to use another way of getting RF into Tasmota
https://github.com/arendst/Tasmota/discussions/13283

There is no guarantee that it will work better

**Reply by sfromis** [reply]
In general, the barebones RF support in Tasmota is focused on simple button style remote controls, with not much being done to try to support other devices like sensors. Not saying that it "can never work", just that you may have to find and integrate the relevant protocol decoder, not directly in Tasmota which is using [rc-switch](https://github.com/sui77/rc-switch) for decoding if/when you hardware modify the new revision of the Sonoff RF bridge. The outdated RF 433 realm is not being worked on by many, last release of rc-switch is about 20 months ago.

**Reply by iointerrupt** [reply]
I was thinking at least raw data would be visible (e.g: ```{"Time":1666374799,"RfRaw":{"Data":"AAA41ED001E8059200F2D255"}}```) even if it cant be "decoded". I  have some pir 433 motion sensors and a sonoff rm433 (8 rf button device), which show up just fine without issues ion rf bridge - by that I mean the raw codes.

Before I go on, full disclosure: im a n00b at this. Received an RTL-SDR 3 days ago and playing with visualizing data using GQRX to try and troubleshoot and have fun with all the neat data flying thru the air :-).  I was visualizing the events using GQRX for button presses, motion sensors and the acurite readings and noticed the following 
![rf_out](https://user-images.githubusercontent.com/7555064/197268882-6c871ef4-6570-4826-ac8c-3eec2f589859.png)

The Sonoff RM433 Buttons when pressed peak at 433.915mhz.  The motion pir sensors (not pictured) peak at ~433.923mhz when activated.  The Acurite at ~433.957mhz. Could it be "out of range" for the rf bridge?  As I am new to this, please forgive if my understanding is off.

**Reply by sfromis** [reply]
Protocol decoding is not only about being able to understand the structure of raw data bytes, but about even being able to transform radio stream data to raw data, there are different ways splitting high/low state/timing into bits, and then into bytes.

And yes, "out of range" may be an issue too, while I do not remember the numbers, my understanding is that the Sonoff RF bridge may be more strict about which channels it tunes into (there are 69 channels from 433.075 MHz to 434.775 MHz), while a SDR dongle is deliberately going for "everything".

Warning: The above may give a *false* impression of me knowing much about how the details work. My experience is that just a simple doorbell push was not picked up by the OG Sonoff RF bridge, but visible using a SDR dongle. A replacement (for other reasons) was picked up by both.
